apps/tests/lib/stackSuccess.ts

```diff
@@ -0,0 +1,21 @@
+import { Stack, StackProps } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as testConstructs from '@sls-mentor/core/tests';
```

Comment:
why not put test constructs inside src ? it would avoid weird inter-package imports

Reply:
my end goal is not to have `stackSuccess` and `stackFail` in a seperate repo in the monorepo. This line will eventually be obsolete. I'm, on the other hand, quite sure index.ts should only export the exposed (understand public) API of `sls-mentor/core` and not everything, to avoid other packages using core internal methods.
Tests constructs do seem like something that should never be exported by the package, especially since all code related will require dev dependencies, which will not be present in the packaged version of core